### PR TITLE
[TEST][WIP] Increase java agent buffer

### PR DESCRIPTION
**Pull request by kuisathaverat**
related to https://github.com/elastic/apm-integration-testing/pull/401

### CODE REVIEW

File: tests/agent/concurrent_requests.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def load_test(self):
         self.logger.debug("Starting tornado I/O loop")
         ioloop.IOLoop.instance().start()
 
-    def check_counts(self, it, max_wait=60, backoff=1):
+    def check_counts(self, it, max_wait=60, backoff=5):
```
Comment (eyalkoren) [inline tests/agent/concurrent_requests.py]: @kuisathaverat You only increased the backoff here, not the wait time. I also looked at this but talked to Silvia and got to the conclusion it is not likely. Also, there is always exactly one missing when they fail, which doesn't seem to fit.

**Comment by eyalkoren** [comment]
Failed differently- `AssertionError: queried for [{'processor.event': 'transaction'}], expected 1375, got 0` 😫

File: tests/agent/concurrent_requests.py (hunk 1 above)
Comment (kuisathaverat) [reply tests/agent/concurrent_requests.py]: I know, I want to not bother Elasticsearch, to see if issues are related to Elasticsearch load, seems not.

**Comment by kuisathaverat** [comment]
I am checking the data https://apm-ci.elastic.co/job/apm-integration-test-axis-pipeline/1799/artifact/tests/results/data-test_concurrent_req_java_spring_foobar.json

**Comment by kuisathaverat** [comment]
There are not transactions, How is it possible?

```
cat tests/results/data-test_concurrent_req_java_spring_foobar.json |jq '.hits.hits[]._source.processor.event'|sort|uniq -c

  12 "metric"
 988 "span"
```

**Pull request closed without merging**